config/initializers/closure_tree_config.rb

```diff
@@ -0,0 +1,5 @@
+ClosureTree.configure do |config|
+  # Some PaaS like Heroku don't have available the db in some build steps like
+  # assets:precompile, this is skipped when this value is true, default = false
+  # config.database_less = true
```

Comment:
Do we need to uncomment this based on the comment above?

Reply:
Possibly